CDCgov/prime-reportstream
# HL7 NK1 Mapping for OML

This PR adds mapping to HL7 `NK1` fields for the `OML` message type

Test Steps:
1. Run `./gradlew primeCLI --args='fhirdata --output-format HL7 --input-file "./src/testIntegration/resources/datatests/FHIR_to_HL7/sample_OML_20230831-0001.fhir"  -s "./metadata/hl7_mapping/receivers/Flexion/TILabOrder.yml"'`
2. Check that the output HL7 message has the `NK1` field

## Changes
- Updated `NK1-3` to use `CWE` schema
- Updated OML input and expected output for integration test

## Checklist

### Testing

## Linked Issues
[Create Outbound OML NK1 Segment](https://github.com/CDCgov/trusted-intermediary/issues/491)
